src/core/agent/services/ipexCommunicationService.test.ts

```diff
@@ -813,6 +813,100 @@ describe("Ipex communication service of agent", () => {
     ).rejects.toThrowError(Agent.KERIA_CONNECTION_BROKEN);
   });
 
+  test("Cannot get linkedGroupRequest from ipex/grant if the notification is missing in the DB", async () => {
+    const id = "uuid";
+    const date = new Date().toISOString();
+    const notification = {
+      id,
+      createdAt: date,
+      a: {
+        d: "d",
+      },
+      connectionId: "EGR7Jm38EcsXRIidKDZBYDm_xox6eapfU1tqxdAUzkFd",
+      read: true,
+    };
+    notificationStorage.findById.mockResolvedValueOnce(null);
+
+    await expect(
+      ipexCommunicationService.getLinkedGroupFromIpexGrant(notification.id)
+    ).rejects.toThrowError(
+      `${IpexCommunicationService.NOTIFICATION_NOT_FOUND} ${id}`
+    );
+  });
+
+  test("Should return accepted and membersJoined when linkedGroupRequests contain valid data", async () => {
+    const id = "uuid";
+    const date = new Date().toISOString();
+    const notification = {
+      id,
+      createdAt: date,
+      a: {
+        d: "d",
+      },
+      connectionId: "EGR7Jm38EcsXRIidKDZBYDm_xox6eapfU1tqxdAUzkFd",
+      read: true,
+    };
+
+    const grantNoteRecord = {
+      linkedGroupRequests: {
+        credentialSaid: {
+          accepted: true,
+          saids: {
+            ipexGrantSaid: [["member"], ["multisigExn"]],
```

Comment:
This is actually the `ipexAdmitSaid` - lets update to avoid confusion. But can you do something like this:
```
ipexAdmitSaid1: [["memberA"], ["multisigExn1A"], ["memberB"], ["multisigExn1B"]],
ipexAdmitSaid2: [["memberA"], ["multisigExn2A"]]
```

The result (based on my other comments) should be `["memberA", "memberB"]`